src/tests/generate_trigger_test.ts

```diff
@@ -0,0 +1,61 @@
+import { generateTriggerFile } from "../generate_trigger.ts";
+import { assert, assertEquals } from "../dev_deps.ts";
+
+Deno.test("generateTriggerFile creates a valid TS file", async () => {
+  const input: string[] = [`--source={ "triggers": \
+    [{ "type": "shortcut", \
+    "name": "test trigger", \
+    "description": "Testing file generation", \
+    "workflow": {"callback_id": "test_trigger"}, \
+     "inputs":{ \
+      "interaction": {"value":"{{data.interactivity}}"}
+      } 
+    }]}`];
+
+  const result = await generateTriggerFile(input);
+  const file = await import(`file://${Deno.cwd() + "/" + result.files[0]}`);
+
+  // Directory exists
+  const dirInfo = await Deno.stat("triggers");
+  assert(dirInfo.isDirectory);
+  // File exists
+  const fileInfo = await Deno.stat(result.files[0]);
```

Comment:
Same thing here: we don't want our unit tests interacting with the filesystem (or, really, doing any kind of I/O outside of memory, in order to keep them fast and the test focussed on testing what matters: the logic of the module-under-test). So, we probably want to stub out the `Deno.writeTextFile` method, and assert that it was called with the right arguments.